lib/osut/utils.rb

```diff
@@ -2090,6 +2090,60 @@ def unconditioned?(space = nil)
     ok
   end
 
+  ##
+  # Validates whether a space can be considered as REFRIGERATED.
+  #
+  # @param space [OpenStudio::Model::Space] a space
+  #
+  # @return [Bool] whether space is considered REFRIGERATED
+  # @return [false] if invalid input (see logs)
+  def refrigerated?(space = nil)
+    mth = "OSut::#{__callee__}"
+    cl  = OpenStudio::Model::Space
+    tg0 = "refrigerated"
+    return mismatch("space", space, cl, mth, DBG, false) unless space.is_a?(cl)
+
+    # 1. First check OSut's REFRIGERATED status.
+    status = space.additionalProperties.getFeatureAsString(tg0)
```

Comment:
Similar to _plenum?_ and _vestibule?_, _refrigerated?_ gives priority to an _AdditionalProperties_ tag (useful e.g. when thermal zoning is in an incomplete state).